### DISCUSSION TITLE
Metroid Dread

### DISCUSSION THREAD

**Discussion by Hamsterer** [skyline-emu/skyline#538] [Game Requests]
![Metroid_Dread_game_box](https://user-images.githubusercontent.com/101499663/159210958-1ca88f8c-ccd1-4bf1-bb62-dbd5975977bd.jpg)

**Comment by Jariel87** [comment]
We have sound on metroid dread with latest built, also fps stable 40-50

![Screenshot_20220331-002048](https://user-images.githubusercontent.com/97302933/160976213-6939f04c-2205-48f8-b192-268d95b938a4.jpg)
 
Good work team

**Comment by sauradipdebnath** [comment]
It boots upto intro. Work on it

**Reply by Jariel87** [reply]
Which built, on my one plus 8 5g only got sound

**Comment by RPYgamers** [comment]
On which build ?

**Comment by Aminemo93** [comment]
Metroid  please!!!!!!!

**Comment by Jariel87** [comment]
I can see videos on youtube with metroid running, dont know what device or processor it is, I have a SD 865 and 8GB ram (oneplus 8 5G), does anyone know?

**Comment by Manevilla** [comment]
Please work on Metroid!
